pkg/cmd/roachtest/tests/admission_control_latency.go

```diff
@@ -219,7 +219,7 @@ func registerLatencyTests(r registry.Registry) {
 	addFull(r, decommission{drain: true}, 5.0)
 	addFull(r, backfill{}, 40.0)
 	addFull(r, &slowDisk{slowLiveness: true, walFailover: true}, math.Inf(1))
-	addFull(r, elasticWorkload{}, 5.0)
+	addFull(r, elasticWorkload{}, 20.0)
```

Comment:
What about other tests, e.g. `addNode` / `decommission`? Are they less susceptible to this flakiness?

Reply:
Those tests are much more reliable at 5.0 (see https://roachperf.crdb.dev/?filter=perturbation&view=perturbation&tab=gce for some examples). Decommission is slightly above addNode, but they are both typically well below the 5.0 increase. The elastic workload is typically 3-4x above and often can get above 5x.